CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 4.2.0 (2022-05-19)
```

Comment:
i think this needs to _technically_ be changed to 2022-05-20 since it didn't get merged in yesterday

Reply:
yes! I'm on it :)